_posts/2016/2016-03-31-ecommerce-barcamp-jena.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: " A report from the eCommerceCamp in Jena"
+date: "2016-03-31 09:03:24"
+icon: shopping-cart
+categories: events
+authors: ["Benjamin"]
+---
+
```

Comment:
Please use one sentence per line (see Markdown style guide). This is easier to read and review.